package.json

```diff
@@ -41,7 +41,7 @@
     "version": "./scripts/update-version.sh ${npm_package_version} && git add CHANGELOG.md"
   },
   "dependencies": {
-    "@mia-platform/lc39": "^3.1.4",
+    "@mia-platform/lc39": "^3.3.0",
```

Comment:
I'd expect a change in the package-lock file was it already using this version? 🤔

Reply:
I think so, I tried to run `npm install` after changing it and nothing changed in the package-lock, should I try it again?

Reply:
my bad, there is no package-lock in this library.